silverback/__init__.py

```diff
@@ -1,6 +1,24 @@
-from .exceptions import CircuitBreaker, SilverbackException
-from .main import SilverbackBot
-from .state import StateSnapshot
+def __getattr__(name: str):
```

Comment:
It doesn't look like you are, but one thing to be aware of is autodoc won'tt work for lazy modules like this.
However, it isn't that hard to just add in the docs manually. Example: https://github.com/ApeWorX/ape/blob/main/docs/methoddocs/ape.md